### PR TITLE
Add ogg theora

### PR DESCRIPTION
**Pull request by chakravarthi589**
@ajpmaclean : Please review.

### CODE REVIEW

File: src/Java/Video/OggTheora.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+
+    String fileName = "test.avi";
+    for(int i = 0; i < args.length; ++i)
```
Comment (ajpmaclean) [inline src/Java/Video/OggTheora.java]: Give the user the opportunity to input a file name. Look at WriteVTP.java and for a suggested name use `OggTheora.avi`

File: src/Java/Video/OggTheora.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    writer.SetInputConnection(source.GetOutputPort());
+    writer.SetFileName(fileName);
+    writer.Start();
+
+    for(int i = 0; i < 100; i++)
+    {
+      source.SetDrawColor(0,0,0,1); //black
+      source.FillBox(0, 100, 0, 100); //clear image
+      source.SetDrawColor(255,0,0,1); //red
```
Comment (ajpmaclean) [inline src/Java/Video/OggTheora.java]: Use vtkNamedColors

**Comment by chakravarthi589** [comment]
please review the updated one.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**